ui.tests/test-module/specs/fragment/fragment.authoring.spec.js

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
Please also add the new components here, https://github.com/adobe/aem-core-forms-components/blob/master/ui.af.apps/pom.xml#L296-L321